Client/Core/Commands/MiscHandler.cs

```diff
@@ -20,11 +20,11 @@ public static partial class CommandHandler
 
             new Thread(() =>
             {
-                string tempFile = FileHelper.GetTempFilePath(".exe");
+                var tempFile = FileHelper.GetTempFilePath(".exe");
```

Comment:
I understand your point. It would just be a good idea to make it the same as it is throughout the codebase of Quasar so we are not inconsistent in certain parts of the codebase.